<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="ANIL" failures="2" tests="2" name="Default test(failed)(failed)" time="0.116" errors="0" timestamp="2023-07-12T14:32:21 IST">
  <testcase classname="failedtestcases.test1" name="demo1" time="0.027">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at failedtestcases.test1.demo1(test1.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- demo1 -->
  <testcase classname="failedtestcases.test1" name="demo4" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at failedtestcases.test1.demo4(test1.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- demo4 -->
  <testcase classname="failedtestcases.test1" name="demo2" time="0.0">
    <ignored/>
  </testcase> <!-- demo2 -->
  <testcase classname="failedtestcases.test1" name="demo3" time="0.0">
    <ignored/>
  </testcase> <!-- demo3 -->
</testsuite> <!-- Default test(failed)(failed) -->
